components/program_manager/db/migrate/20210812163711_add_indexes_to_transactions.rb

```diff
@@ -0,0 +1,13 @@
+class AddIndexesToTransactions < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    change_table :program_manager_transactions do |t|
+      # As PostgreSQL manages to combine indexes, I didn't feel the need to add
+      # composite indexes
+      t.index :customer_id
```

Comment:
We actually don't need the `customer_id` since a transaction belongs to a card and a card belongs to a customer.

```suggestion
```